html/pfappserver/root/static.alt/src/views/Configuration/_components/AdminRoleView.vue

```diff
@@ -144,22 +148,29 @@ export default {
         }
       })
     },
+    remove (item) {
+      this.$store.dispatch('$_admin_roles/deleteAdminRole', item.id).then(response => {
+        this.close()
+      })
+    },
     save () {
       const ctrlKey = this.ctrlKey
       this.$store.dispatch(`${this.storeName}/updateAdminRole`, this.form).then(response => {
         if (ctrlKey) { // [CTRL] key pressed
           this.close()
         }
       })
-    },
-    remove () {
-      this.$store.dispatch(`${this.storeName}/deleteAdminRole`, this.id).then(response => {
-        this.close()
-      })
     }
   },
   created () {
     this.init()
+  },
+  watch: {
+    isClone: {
```

Comment:
Could we do this check in `mounted` instead? I don't think a watcher is necessary since `isClone` won't changed.